content/en/blog/_posts/2021-07-19-alpha-swap-support.md

```diff
@@ -0,0 +1,139 @@
+# Node alpha swap memory support
+
+---
+layout: blog
+title: 'Alpha swap support'
+date: 2021-07-19
+slug: alpha-swap-support
+---
+
+**Author:** Elana Hashman (Red Hat)
+
+The 1.22 release introduces alpha support for configuring swap memory usage for
```

Comment:
nit:
As this will go out post release, I suggest:
```suggestion
The 1.22 release introduced alpha support for configuring swap memory usage for
```